client/src/components/UploadImages.js

```diff
@@ -0,0 +1,96 @@
+import React, { useCallback } from 'react';
+import { withStyles, Typography } from '@material-ui/core';
+import { useDropzone } from 'react-dropzone';
+import Image from './Image';
+
+const styles = {
+    uploadImagesContainer: {
+        marginTop: '30px',
+        marginBottom: '10px'
+    },
+    dropzoneContainer: {
+        display: 'flex',
+        marginTop: '20px',
+        marginBottom: '20px',
+    },
+    dragBox: {
+        display: 'flex',
+        boxShadow: '0px 0px 3px 2px #D3D3D3',
+        height: '150px',
+        width: '175px',
+        margin: '2px',
+        padding: '5px',
+        cursor: 'pointer',
+        borderRadius: '2px',
+        outline: 'none'
+    },
+    dragBoxText: {
+        margin: 'auto',
+        textAlign: 'center',
+    },
+    uploadImageIcon: {
+        height: '40px',
+        width: '40px',
+    },
+    images: {
+        display: 'flex',
+        flexWrap: 'wrap',
+        height: '150px',
+        width: '250px',
+        margin: 'auto',
+    },
+    // image: {
+    //     maxHeight: '75px',
+    //     maxWidth: '75px',
+    //     padding: '4px',
+    //     borderRadius: '10px',
+    //     cursor: 'pointer'
+    // }
+}
+
+const UploadImages = (props) => {
+    const { classes, setImages, images, deleteImage } = props;
+
+    const onDrop = useCallback(acceptedFiles => {
```

Comment:
I'm totally nitpicking but useCallback could use a more descriptive name.

Reply:
it's actually a react hook so I don't think I can do anything about it. I also used it from the example provided in the react drop-zone docs. I think useCallback() returns a memoized callback so it's more efficient when dealing with these images

Reply:
Got it. I'm just going to go ahead and relay my feedback to the drop-zone people...

Reply:
it's from the react hook devs! loll